gatling-ss7/src/main/scala/io/snice/gatling/ss7/engine/Ss7Callbacks.scala

```diff
@@ -0,0 +1,36 @@
+package io.snice.gatling.ss7.engine
+
+import com.typesafe.scalalogging.StrictLogging
+import io.gatling.core.Predef.Status
+
+import scala.collection.mutable
+
+object Ss7Callbacks extends StrictLogging {
+  class RequestId(val transactionId: Long, val invokeId: Long) {
```

Comment:
this inner class looks a bit weird, since you are using it also outside of this object. i would probably take it out